.gitignore

```diff
@@ -102,3 +102,4 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+.idea
```

Comment:
shouldn't we put the folder on the `.gitignore` for all sub folders too?
if I remembered it correctly, last time the `mains/.idea` folder was uploaded.